### PR TITLE
Separate JS from HTML in code snippets

### PR DESCRIPTION
**Pull request by dorlinski**
Signed-off-by: Dominik Orliński <d.orlinski@samsung.com>

### CODE REVIEW

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm [hunk 1]
```diff
@@ -284,15 +286,20 @@ <h3>Summary</h3>
...
+                                prettyprint">
+var navigation = page.querySelector(&quot;#navigation&quot;),
+    activePanel = page.querySelector(&quot;.ui-panel-active&quot;),
+    navigationComponent = tau.widget.Navigation(navigation);
+navigationComponent.push(activePanel.id);
+</pre>
             </div>
 
 
+
```
Comment (lslachciak) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: unneded line?

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm [hunk 2]
```diff
@@ -321,21 +328,31 @@ <h3>Summary</h3>
...
+<pre name="code" class="examplecode prettyprint">
+var navigation = page.querySelector(&quot;#navigation&quot;),
+    navigationComponent = tau.widget.Navigation(navigation);
+
+navigationComponent.pop();
 </pre>
             </div>
 
 
+
```
Comment (lslachciak) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: ditto.

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm (hunk 1 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm (hunk 2 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm]: Fixed

**Pull request merged**